config/default.js

```diff
@@ -331,6 +331,8 @@ module.exports = {
   // because they are app-independant, like `/en-US/developers/`.
   validClientAppUrlExceptions: [
     '__frontend_version__',
+    '__heartbeat__',
+    '__lbheartbeat__',
```

Comment:
I am thinking that this is going to conflict with addons-server. We might want to have `__frontend_heartbeat__` and `__frontend_lbheartbeat__` but we need to ask @bqbn to see if that'd be OK.

Reply:
I got the impression these would be called on the underlying instances so the nginx config won't be used (could be wrong).

Reply:
@bqbn Can you comment on this, please?

Reply:
`__lbheartbeat__` and `__heartbeat__` are fine because these are to be used by the ELBs to check the application's health. No Nginx is involved.

In the `__frontend_version__` case, that's because we needed a way to check the frontend's version from outside and that had to go through Nginx, since `__version__` is taken by the addons-server, thus we needed a new path that's called `__frontend_version__`.

Reply:
Oh, actually, strictly speaking only `__lbheartbeat__` is used by the ELBs, we may make other use of `__heartbeat__` in the future. 🤷 

So for naming consistency purpose, feel free to make them `__frontend_heatbeat__` and `__frontend_lbheartbeat__`. Or if you want to create them as aliases, feel free to do that too.

Reply:
> Oh, actually, strictly speaking only `__lbheartbeat__` is used by the ELBs, we may make other use of `__heartbeat__` in the future. 🤷
> 
> So for naming consistency purpose, feel free to make them `__frontend_heatbeat__` and `__frontend_lbheartbeat__`. Or if you want to create them as aliases, feel free to do that too.

We can call them whatever you need us to, but I'm not clear from above exactly what you need. Is it fine if we *just* call them `__frontend_heatbeat__` and `__frontend_lbheartbeat__` and not have `__heartbeat__` or `__lbheartbeat__`, or do you need all 4 of them defined for frontend?

Reply:
Call them `__frontend_heartbeat__` and `__frontend_lbheartbeat__` please.